MediaGalleryUi/Model/GetDetailsByAssetId.php

```diff
@@ -102,9 +102,11 @@ public function execute(int $assetId): array
      */
     private function getContentType(string $contentType): string
     {
-        return strpos($contentType, '/') !== false
+        $result = strpos($contentType, '/') !== false
             ? substr($contentType, strpos($contentType, '/') + 1)
             : $contentType;
+
+        return strtoupper($result);
```

Comment:
```suggestion
       return strtoupper(str_replace('image/', '', $contentType)
```

Reply:
@engcom-Charlie we had a talk with @Nazar65 about this change request. In your implementation, everything good but I would ask you to implement the suggestion, please.